manager/state/raft/raft.go

```diff
@@ -1681,7 +1689,7 @@ func (n *Node) processInternalRaftRequest(ctx context.Context, r *api.InternalRa
 
 	// Do this check after calling register to avoid a race.
 	if atomic.LoadUint32(&n.signalledLeadership) != 1 {
-		log.G(ctx).Errorf("node %x is no longer leader, aborting propose", n.opts.ID)
+		log.G(ctx).Error("node is no longer leader, aborting propose")
```

Comment:
@anshulpundir why remove node IDs from these log messages?

Reply:
node id is a field already added as a field to the contest logger https://github.com/docker/swarmkit/blob/master/node/node.go#L286

Reply:
node id is a field already added as a field to the contest logger https://github.com/docker/swarmkit/blob/master/node/node.go#L286 @nishanttotla

Reply:
Ah, okay got it.